packer/ubuntu16.json

```diff
@@ -0,0 +1,41 @@
+{
+    "variables": {
+        "my_machine_type": "e2-medium",
+        "network": "projects/clgcporg2-057/global/networks/default",
+        "diskSizeGb": "20",
+        "vm_name": "instance-abc", 
+        "diskTypes": "pd-balanced"
+    },
+    "builders": [
+        {
+            "type": "googlecompute",
+            "project_id": "{{user `project_id`}}",
+            "image_name": "reddit-full-{{timestamp}}",
+            "image_family": "reddit-full",
```

Comment:
Looks like this file should describe how to build `reddit-base` template rather than `reddit-full`